Patch to allow us to specify an alternate indent binary when running tests.

This patch should be sent upstream.

--- indent-2.2.9/regression/TEST.orig	2015-08-03 07:55:36.236058899 -0700
+++ indent-2.2.9/regression/TEST	2015-08-03 07:56:32.264810064 -0700
@@ -3,7 +3,7 @@
 # Compare the output of ../src/indent to correct output
 # generated by earlier versions
 
-INDENT=../src/indent
+INDENT="${INDENT_BIN:-../src/indent}"
 
 # First, make sure the executable exists
 
